app/boards/intel_adsp_ace30_ptl.conf

```diff
@@ -45,7 +45,6 @@ CONFIG_ZEPHYR_NATIVE_DRIVERS=y
 CONFIG_INTEL_MODULES=y
 CONFIG_LIBRARY_MANAGER=y
 CONFIG_LIBRARY_BASE_ADDRESS=0xa0688000
-CONFIG_LIBRARY_BUILD_LIB=y
```

Comment:
Don't think this is the agreed strategy for PTL. We do want a library build instead of dozens of individual module files. If this is a temporarily change for a test, please mark this as such to avoid accidental merging

Reply:
Please keep it, this PR #9841 will be needed to cover AEC case + CONFIG_LIBRARY_BUILD_LIB=y